tests/unittest_for_generator.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import os
+import sys
+import inspect
+current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parent_dir = os.path.dirname(current_dir)
+sys.path.insert(0, parent_dir)
+import range_generator
+from range_generator import *
+
+
+class TestGenerator(unittest.TestCase):
+
+    def test_MyRange(self):
+        generator = range_generator.MyRange(0, 10, 2)
+        self.assertEqual(0, next(generator))
+        self.assertEqual(2, next(generator))
+        self.assertEqual(4, next(generator))
+        self.assertEqual(6, next(generator))
+        self.assertEqual(8, next(generator))
+
+        generator = range_generator.MyRange(0, 100, 10)
+        a = 0
+        for i in range(10):
+            a = next(generator)
+        self.assertEqual(90, a)
```

Comment:
Does it worth to do the same check on every cycle step?

Reply:
Now, I check only the last step. Do you mean I should check matches on every step?

Reply:
I would say yes.